tests/acme_diags/derivations/test_acme.py

```diff
@@ -0,0 +1,80 @@
+from typing import TYPE_CHECKING
+from unittest import TestCase
+from unittest.mock import Mock
+
+import numpy as np
+
+from acme_diags.derivations.acme import adjust_prs_val_units, determine_cloud_level
+
+if TYPE_CHECKING:
+    from cdms2.axis import FileAxis
+
+
+class TestAdjustPrsValUnits(TestCase):
+    def setUp(self):
+        self.mock_file_axis: "FileAxis" = Mock()
```

Comment:
This creates a mock object to mock `cdms2.axis.FileAxis`.

More info here: on the `unittest.mock` API: https://docs.python.org/3/library/unittest.mock.html

>Mocking is a process used in unit testing when the unit being tested has external dependencies. The purpose of mocking is to isolate and focus on the code being tested and not on the behavior or state of external dependencies. 
> &mdash; <cite>https://www.telerik.com/products/mocking/unit-testing.aspx#:~:text=Mocking%20is%20a%20process%20used,or%20state%20of%20external%20dependencies.</cite>